services/slam/builtin/cartographer_int_test.go

```diff
@@ -165,6 +171,10 @@ func integrationTestHelperCartographer(t *testing.T, mode slam.Mode) {
 	for {
 		line, err := logReader.ReadString('\n')
 		test.That(t, err, test.ShouldBeNil)
+		if strings.Contains(line, "Passed sensor data to SLAM") {
+			// Check delete_processed_data is working as intended
```

Comment:
[nit] I think you can remove these comments, since they are the same as the function name.